tests/integration/assessment/test_external_locations.py

```diff
@@ -0,0 +1,30 @@
+import logging
```

Comment:
why this file is in `assessment` folder? it has to be in `hive_metastore`. please fix.